applications/GeoMechanicsApplication/python_scripts/geomechanics_solver.py

```diff
@@ -21,8 +21,8 @@ def __init__(self, model, custom_settings):
 
         super().__init__(model, custom_settings)
 
-        # # Overwrite the default settings with user-provided parameters.
-        # self.settings.ValidateAndAssignDefaults(default_settings)
+        # Validate settings.
```

Comment:
I think this comment is redundant. The function name (`ValidateSettings`) says it all 😃